artemis-journal/src/main/java/org/apache/activemq/artemis/core/journal/impl/JournalImpl.java

```diff
@@ -2625,12 +2631,13 @@ private void scheduleCompact() {
          return;
       }
 
+      logger.debug("JournalImpll::scheduleCompact");
```

Comment:
Just checking its intended...this one is debug when all the others added are trace?

Reply:
I will keep it as debug, and moved others to debug